bitnami/kafka/3.5/debian-11/rootfs/opt/bitnami/scripts/libkafka.sh

```diff
@@ -978,7 +978,13 @@ kafka_initialize() {
             listener_lower="$(echo "$listener" | tr '[:upper:]' '[:lower:]')"
 
             if [[ "$protocol" = "SSL" || "$protocol" = "SASL_SSL" ]]; then
-                kafka_server_conf_set "listener.name.${listener_lower}.ssl.client.auth" "$KAFKA_TLS_INTER_BROKER_AUTH"
+                if [[ "$listener" = "${KAFKA_CFG_INTER_BROKER_LISTENER_NAME:-INTERNAL}" ]]; then
+                    kafka_server_conf_set "listener.name.${listener_lower}.ssl.client.auth" "$KAFKA_TLS_INTER_BROKER_AUTH"
+                elif [[ "${KAFKA_CFG_CONTROLLER_LISTENER_NAMES:-CONTROLLER}" =~ $listener ]]; then
```

Comment:
Please use the same the syntax than in the first if

```suggestion
                elif [[ "$listener" = "${KAFKA_CFG_CONTROLLER_LISTENER_NAMES:-CONTROLLER}" ]]; then
```

Reply:
> Please use the same the syntax than in the first if

I used that format in order to be consistent with how the controller listener is detected on line 995:
`elif [[ "${KAFKA_CFG_CONTROLLER_LISTENER_NAMES:-CONTROLLER}" =~ $listener ]]; then`
My assumption is it was done that way because the config is a comma separated list rather than just a single name.

In light of that, would you still like me to change it?